### DISCUSSION TITLE
where is User Role Permissions - Advanced Options & Configurations

### DISCUSSION THREAD

**Discussion by yureckey** [nocodb/nocodb#3533] [General]
I'm looking at latest release  0.96.0 (windows executable) and I'm trying to find where is
User Role Permissions - Advanced Options & Configurations 
which I read about in documentation:
https://docs.nocodb.com/setup-and-usages/team-and-auth#advanced-options--configurations
I cannot find it (

**Comment by wingkwong** [comment]
The table in documentation only shows what options a role could do. Currently you cannot change it. However, you can change UI Access Control in Team & Settings > Project Metadata > UI Access Control just like previous version.

**Reply by yureckey** [reply]
ah, I see, I thought it's description of existing UI for advanced permissions.
UI access controls - yes, it allows to hide certain tables. I wish there would be more options for fine-tuning (separate controls for Add/Delete/Update rows)
